src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -935,5 +935,48 @@ private static ErrorValue GetDiv0Error(IRContext irContext)
                 Message = "Division by zero"
             });
         }
+
+        private static FormulaValue Dec2Hex(IRContext irContext, NumberValue[] args)
+        {
+            var number = Math.Floor(args[0].Value);
+            var places = Math.Floor(args[1].Value);
+
+            if (double.IsInfinity(number) || double.IsInfinity(places))
```

Comment:
valid range in XL: -549755813888 ... +549755813887
Or 0x8000000000 .. 0x7FFFFFFFFF